### ISSUE TITLE
Installation on Windows fails if gofish is already installed

### ISSUE THREAD

**Issue by notlmn** [fishworks/gofish#57]
Gofish cannot be installed (updated) with gofish because of #46.

But gofish also fails to install on Powershell as it refuses to replace gofish binary that already exits!

```
PS C:\Windows\system32> iex ((New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/fishworks/gofish/master/scripts/install.ps1'))
Downloading https://gofi.sh/releases/gofish-v0.4.0-windows-amd64.zip
Preparing to install into C:\ProgramData\bin
Move-Item : Cannot create a file when that file already exists.
At line:23 char:1
+ Move-Item -Path "$tempDir\windows-amd64\gofish.exe" -Destination "$in ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : WriteError: (C:\Users\xxxxxx...md64\gofish.exe:FileInfo) [Move-Item], IOException
    + FullyQualifiedErrorId : MoveFileInfoItemIOError,Microsoft.PowerShell.Commands.MoveItemCommand

gofish installed into C:\ProgramData\bin\gofish.exe
Restart your terminal, then run 'gofish init' to get started!
PS C:\Windows\system32>
```

**Comment by notlmn** [comment]
This should be a separate issue but I tried to download and install gofish, but Chrome thinks its harmful.

![image](https://user-images.githubusercontent.com/37769974/38778860-3bf5f2c4-40dd-11e8-9b07-1ed142049ed8.png)

------------------

I've even uploaded the archives to Virus Total and one engine suspects Windows zip archive, but not other archives.
* [Windows binary in zip](https://www.virustotal.com/#/file/069ac4806f513bb2d0792583a8ed49c2cbaa46b4d38330f70ce9480211a78eeb)
* [Windows binary in tar](https://www.virustotal.com/#/file/8d0353cd4c56dfb0477c7de028ee924fe2812bdcd4830289ab714b649ed8fc71)
* [Linux binary](https://www.virustotal.com/#/file/cc415b6b5c7f73534f9fe73564ea70ca23529d5e81fd09146a85658437b02510)
* [macOS binary](https://www.virustotal.com/#/file/a04363917b20474395ae1617e10260d063ca5cb53d8441ebf7b9f2c437f74e20)

-------

Can this not be made `go get`able?

**Comment by bacongobbler** [comment]
Yeah so the first thing is definitely a bug in the PowerShell script. Feel like taking a crack at it?

That's interesting that it's just the zip archive that's considered unsafe.

Gofish is `go get` able, but I'd rather users actually use the release binaries instead because

1. that's been guaranteed to be tested for quality assurance
2. I cannot guarantee that the dependencies gofish relies on will always work on master

If you'd like to build from source, there's the [hacking guide](https://www.gofi.sh/#hacking) which follows through the steps to build from source.

**Comment by notlmn** [comment]
> Yeah so the first thing is definitely a bug in the PowerShell script. Feel like taking a crack at it?

#58

> `go get` IMO is a horrible, *horrible* release tool due to its nature of fetching everything off master

Wonder why they ever designed it that way. I'm no Go programmer but what about versioned branches?
